[ADD] product_multi_price: New Module (#595)
OCA/product-attribute: product_multi_price/README.rst

==================

> Product Multi Cost

Reviewer: ```suggestion
Product Multi Price
```

.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
    :target: https://github.com/OCA/product-attribute/tree/11.0/product_multi_price
    :alt: OCA/product-attribute
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/product-attribute-11-0/product-attribute-11-0-product_multi_price
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/135/11.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

This module allows to set multiple prices to products and base pricelist
rules on them.

**Table of contents**

.. contents::
   :local:

Configuration
=============

To configure multiple prices you need to set multi prices field names first.
To do so, you need admin permissions. Then go to:

#. *Settings > General Settings > Technical > Database Structure >
   Price Field Names*
#. Create the multi price fields you need.

Usage
=====

To use this module, you need to:

#. Go to the product page.

> #. In the general tab, there's a list called *Multiple Prices*.

Reviewer: Better to call it just `Prices`
```suggestion
#. In the general tab, there's a list called *Prices*.
```

#. You can add one for every multiple price field name available.

Known issues / Roadmap
======================

* Add mechanisms that allow to set multiprices values from external flows. For

>   example: having AVCO, FIFO and Standard price in diferent fields.

Reviewer: ```suggestion
  example: having AVCO, FIFO and Standard prices computed simultaneously in this table.
```
